### DISCUSSION TITLE
How to glob import scss files in webpack 5

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#13000] [Q&A]
I am trying to glob import all my scss files in the project to the main file named main.scss.

In webpack-3 things worked just fine as I was using the loader import-glob and my sass-loader is given the paths to load the scss files from.

But unfortunately the same setup never works for webpack 5. 

My scss files complain about missing scss functions.

**Comment by Jakobud** [comment]
If your SCSS files are complaining about missing SCSS functions then it sounds like your SCSS files are not importing other SCSS files properly. How are those files imported? Using aliases or just regular functions? If you just use `node-sass` on the command line to build your SCSS files does it work fine but just doesn't work in Webpack?

Also, you might not want to glob ALL your SCSS files into Webpack. Typically you would not want any SCSS files starting with underscore "_" to be globbed in as those would be imported by your other SCSS files.
